docs/references/protocol/transactions/types/ledgerstatefix.md

```diff
@@ -0,0 +1,68 @@
+---
+seo:
+    description: Repair corruptions to the XRP ledger.
+labels:
+  - Utilities, troubleshooting
+---
+# LedgerStateFix
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/LedgerStateFix.cpp "Source")
+
+
+`LedgerStateFix` is a general purpose transaction used to fix specific issues affecting the XRP ledger. You submit the transaction with the `LedgerFixType` value set to indicate the particular  error state to correct.
+
+## Example {% $frontmatter.seo.title %} JSON
+
+```json
+{
+   "Account" : "<Signer and fee payer>",
+   "Fee" : "2000000",
+   "LedgerFixType" : 1,
+   "Owner" : "<Account with corrupted NFTokenPage directory",
+   "Sequence" : <n>,
+   "SigningPubKey" : "<Account's public key>",
+   "TransactionType" : "LedgerStateFix",
+   "TxnSignature" : "<Signature>",
+}
+```
+
+{% raw-partial file="/docs/_snippets/tx-fields-intro.md" /%}
+
+| Field | Data Type  | Required? | Description |
+|:------|:-----------|:----------|:------------|
+| `TransactionType` | uint16 | Required | Identifies this as a `LedgerStateFix` transaction. |
+| `Account` | STAccount | Required | Identifies the account signing and submitting the transaction as well as paying the Fee. |
+| `Fee` | STAmount | Required | This transaction is rare and potentially compute intensive. The minimum fee is the same as the fee for an AccountDelete transaction. If the transaction fails with a tec code, the fee is still charged. |
+| `Flags` | uint32 | Optional | Not needed for `LedgerFixType` == _1_. Reserved for a future type of ledger fix. |
+| `LedgerFixType` | uint16 | Required | Currently the only type is _1_, which fixes the NFToken directory for a single account. |
+| `Owner` | STAccount | Optional | Required if `LedgerFixType` == _1_, the account ID that owns the NFToken directory that needs fixing. Need not have any relationship to Account. |
+
+## LedgerStateFix Flags
+
+Transactions of the LedgerStateFix type can support additional values in the `Flags` field. Currently, there are no flags defined. A future `LedgerFixType` might require flag settings.
+
+## Error Cases
+
+Potential errors are those that can occur for all transactions. {% $frontmatter.seo.title %}. 
```

Comment:
Stray variable here.

```suggestion
Potential errors are those that can occur for all transactions.
```

Also I suspect there's at least one error type that doesn't occur for all transactions (if you try to fix something that isn't broken)